### ISSUE TITLE
Library distribution 

### ISSUE THREAD

**Issue by petrnohejl** [strvcom/android-chat-component#48]
I think that distribution via Bintray would be better than Jitpack for 2 reasons: developers don't have to explicitly declare Maven repository and we would avoid "com.github" prefix in the package name, "com.strv" package looks more professional.

**Comment by kotoMJ** [comment]
Good point noticed Jake's opinion, that you should use package name of domain, you actually own! 
That means you wouldn't use others packages to cover your app.
That should be forced somehow on some distribution places. But I guess Bintray is not forcing this so far. Anyway beware of this.
